catalogs/sources/gtfs/schedule/us-chicago-the-otter-express-gtfs-2038.json

```diff
@@ -0,0 +1,25 @@
+{
+    "mdb_source_id": 2038,
+    "data_type": "gtfs",
+    "provider": "The Otter Express",
+    "name": "Otter Ecpress Medical Transportation Flex v2",
+    "feed_contact_email": "Kaylas@Productivemn.org",
```

Comment:
@AlfredNwolisa Not sure where this feed contact email came from - it looks like the feed_info.txt has a trillium email

Reply:
@emmambd this was the email on otter express website. I will update it to trillium's.

Reply:
@emmambd updates done kind review